src/frontend/packageJson.test.ts

```diff
@@ -37,6 +40,13 @@ describe('frontend package.json', () => {
       expect(semver.satisfies(frontendVersion, backendVersion)).toBe(true);
     }
   });
+
+  it('all front end dependencies use exact version', () => {
+    for (const version of Object.values(allDependencies)) {
+      const isExact = semver.valid(version) !== null;
+      expect(isExact).toBe(true);
+    }
+  });
```

Comment:
given this test, i think you can remove the test that does this for the mapeo-related frontend deps (the test with the title `"uses exact versions for @mapeo dependencies"`)

Reply:
https://github.com/digidem/comapeo-mobile/pull/893/commits/7993eb9ff21ad00e84957db58012f9a637f3f772